pkg/kubectl/cmd/get/humanreadable_flags.go

```diff
@@ -14,13 +14,15 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package printers
+package get
```

Comment:
@soltysh I moved these files to their own package under `cmd`, as I needed to break an import cycle when depending on `printersinternal.AddHandlers` below

Reply:
I wonder if we can get rid of it...